raiden/transfer/state.py

```diff
@@ -316,8 +318,16 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
 
     @classmethod
     def from_dict(cls, data: typing.Dict[str, typing.Any]) -> 'ChainState':
+        pseudo_random_generator = random.Random()
+
+        # JSON serializes a tuple as a list
+        state = list(data['pseudo_random_generator'])  # copy
```

Comment:
You do the same here as you do [here](https://github.com/raiden-network/raiden/pull/2399/files#diff-849875864d17875ce53a5c3f63b8b6d7R372). Perhaps factor it out in a util function?